### PR TITLE
add walking skeleton

### PR DESCRIPTION
**Pull request by kudukacar**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```
Comment (damonkelley) [inline Gemfile]: I think we can remove this.

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```
Comment (damonkelley) [inline Gemfile]: We likely won't need this either.

File: lib/tictactoe.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class TicTacToe
+  def run
+    puts "Welcome to Tic-Tac-Toe"
+    true
+  end
+end
+
+if $PROGRAM_NAME == __FILE__
+  TicTacToe.new.run
+end
```
Comment (damonkelley) [inline lib/tictactoe.rb:10]: Alternatively, we could put this into a script in the root of the repo or in a `bin` directory. This is ok too though.

Just an FYI though. No changes needed here :)

**Review by damonkelley — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (kudukacar) [reply Gemfile]: will do

File: Gemfile (hunk 2 above)
Comment (kudukacar) [reply Gemfile]: Sounds good

**Review by damonkelley — APPROVED** [review]
:shipit:

**Pull request merged**